extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -63,13 +68,16 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     const screen = global.screen;
     const slotRect = this.getIntersectionElementLayoutBox();
     const visibilityState = documentStateFor(global).getVisibilityState();
+    const adTestOn = this.element.getAttribute('data-adtest') ||
+        containsExperimentId(this.element.getAttribute(EXPERIMENT_ATTRIBUTE),
```

Comment:
how about passing `this.element` to the function? So that `EXPERIMENT_ATTRIBUTE` and `MANUAL_EXPERIMENT_ID` can be private.

Then you can rename `containsExperimentId` to something like `isInExperiment`